src/containers/GameContainer/GameContainer.tsx

```diff
@@ -0,0 +1,87 @@
+import { BorderGame, Game, MainHero, TimeController } from 'game';
+import { useEffect, useRef } from 'react';
+
+export const GameContainer = () => {
+  const ref = useRef(null);
+
+  useEffect(() => {
+    const canvas = ref.current as HTMLCanvasElement;
+    const width = 1024;
+    const height = 640;
+    if (!canvas) {
+      return;
+    }
+    const ctx = canvas.getContext('2d');
+    if (!ctx) {
+      return;
+    }
+    canvas.width = width;
```

Comment:
❗️
Измерения канваса лучше хранить в константах, можно воспринимать это как часть конфига игры
```suggestion
    const height = 640;
    
    if (!canvas) {
      return;
    }
    
    const ctx = canvas.getContext('2d');
    
    if (!ctx) {
      return;
    }
    
    canvas.width = width;
```

Reply:
Это часть конфига игры. Я рамку подсветки задаю относительно этих данных и пока не понимаю, как делать ресайз)

Reply:
Насчёт ресайза пока не заморачивайся, пусть работает в одном разрешении, сейчас главное все механики прикрутить и насколько хватит времени потюнить